samples/Basic YARP Sample/appsettings.json

```
@@ -0,0 +1,56 @@
+{
+  // Base URLs the server listens on, must be configured independently of the routes below
+  "Urls": [
+    "http://localhost:5000",
+    "https://localhost:5001"
+  ],
```

Urls is one string with semicolons, not an array.
```suggestion
  "Urls": "http://localhost:5000;https://localhost:5001",
```

---

I tried a single string and it threw, I think logically its an array. The array works!.

---

I just confirmed `"Urls": "https://localhost:5004;http://localhost:5002",` works. When I try the array format you have here it's ignored and it uses the default values (which also happen to be 5000 and 5001).

You're right an array would make more sense in the JSON scenario, but this key was designed mainly for use on command line and environment variables.